Formula/lgeneral.rb

```diff
@@ -0,0 +1,38 @@
+class Lgeneral < Formula
+  desc "Turn-based strategy engine heavily inspired by Panzer General"
+  homepage "https://lgames.sourceforge.io/LGeneral/"
+  url "https://downloads.sourceforge.net/lgeneral/lgeneral/lgeneral-1.4.4.tar.gz"
+  sha256 "0a26b495716cdcab63b49a294ba31649bc0abe74ce0df48276e52f4a6f323a95"
+  license "GPL-2.0-or-later"
+
+  depends_on "gettext"
+  depends_on "sdl12-compat"
+  depends_on "sdl2"
+
+  def install
+    inreplace "configure", "#include <unistd.h>", "#include <sys/stat.h>\n#include <unistd.h>"
```

Comment:
Has this been submitted upstream?

Reply:
Not yet. Have a plans, but not yet sorted out sourceforge source control community process.

Reply:
https://sourceforge.net/p/lgeneral/patches/4/

Reply:
Can you add a comment saying something like "Applied in <url>, remove in next release"

Reply:
done